backend/src/email/middlewares/email-transactional.middleware.ts

```diff
@@ -92,11 +162,22 @@ export const InitEmailTransactionalMiddleware = (
         message: 'Rate limited request to send transactional email',
         userId: req?.session?.user.id,
       })
+      // think we can use void here? not using return value, no need to wait
+      void EmailMessageTx.update(
+        {
+          status: TransactionalEmailMessageStatus.RateLimitError,
+          errorCode: '429',
+        },
+        {
+          where: { id: req.body.emailMessageTxId },
+        }
+      )
       res.sendStatus(429)
```

Comment:
Can we update this to `.json({ message: 'Too many requests. Please try again later.' })`

Reply:
ok done